.grit/patterns/sql_alchemy_v2.md

```diff
@@ -47,9 +47,22 @@ pattern convert_to_subquery() {
     } => `$var = select($args).subquery()`
 }
 
+// this pattern can probably be more general
+pattern c_to_selected_columns() {
+    `$x.where($args)` where { 
+        $args <: contains bubble `$a.c.$b` => `$a.selected_columns.$b`
```

Comment:
Isn't this the one changed in the last iteration or am I crazy?

Reply:
I'm bad at staking prs